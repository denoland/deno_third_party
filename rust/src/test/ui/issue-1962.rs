// Copyright 2012 The Rust Project Developers. See the COPYRIGHT
// file at the top-level directory of this distribution and at
// http://rust-lang.org/COPYRIGHT.
//
// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
// option. This file may not be copied, modified, or distributed
// except according to those terms.

// compile-flags: -D while-true
fn main() {
  let mut i = 0;
  while true  { //~ ERROR denote infinite loops with `loop
    i += 1;
    if i == 5 { break; }
  }
}
