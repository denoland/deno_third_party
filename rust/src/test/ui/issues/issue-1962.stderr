error: denote infinite loops with `loop { ... }`
  --> $DIR/issue-1962.rs:4:3
   |
LL |   while true  { //~ ERROR denote infinite loops with `loop
   |   ^^^^^^^^^^ help: use `loop`
   |
   = note: requested on the command line with `-D while-true`

error: aborting due to previous error

